packages/web-components/src/player-component/player.class.ts

```diff
@@ -19,110 +19,12 @@ import { Localization } from './../../../common/services/localization/localizati
 import { IDictionary } from '../../../common/services/localization/localization.definitions';
 import { MediaApi } from '../../../common/services/media/media-api.class';
 import { MimeType } from './player-component.definitions';
+import * as shakaRtsp from '@azure/video-analyzer-player';
```

Comment:
Why call this shakaRtsp,,, may be AvaPlayer or deconstruct the actual interfaces we need.

Reply:
@JangaNA import { ShakaErrorHandler } from '@azure/video-analyzer-player' should work now right?

Reply:
Fixed. @giakas do we still want to hold off on this change since we are only using the new error handler, not the entire new player?